interfaces/xenmgr_vm.xml

```diff
@@ -554,7 +605,6 @@
       <tp:docstring>Extra xenvm arguments separated by semicolon.</tp:docstring>
     </property>
     <property name="flask-label" type="s" access="readwrite"/>
-    <property name="gpu" type="s" access="readwrite"/>
```

Comment:
I think to support gpu passthrough we want to keep this property and the associated methods in manager.

Reply:
Actually this property is still wired up in [xenmgr](https://github.com/OpenXT/manager/pull/180/files#diff-3c747c519f6329b9b8665f5dc997c48eb2ce98adc0e54c74bb6dee8aa1c043b4R818), so lets bring it back.

It would be tremendously helpful to define what's supposed to be in there though. As it is, [disman](https://gitlab.com/vglass/disman/-/blob/master/disman.cpp#L149) seems to use it as a boolean, so the type should be changed in consequence. I *think* the string type is legacy from surfman's deprecated `HDX` (see https://github.com/OpenXT/manager/pull/180).

Reply:
Further down, it says "long form bdf - 0000:00:02.0".  https://gitlab.com/vglass/disman/-/blob/master/disman.cpp#L198

Reply:
This is used in SV to support PT GPUs. It contains the long form bdf of the PT GPU. Disman uses this value if it exists to wire up communications with the guest so as to get a list of displays on the GPU inside the guest.